template-helpers/footer.js

```diff
@@ -28,6 +33,21 @@ function getFooterLinks(args) {
     },
   ];
 
+  // Growth Experiment: Only add the footer share line if user is on VB branch
+  const experimentFlags = getExperimentFlags(args.data.root.req, EXPERIMENTS_ENABLED);
+  if (EXPERIMENTS_ENABLED && experimentFlags.treatmentBranch) {
```

Comment:
This appends the share link to the footer (same in the header too) without breaking the current loop-build flow to hard-code this entry in.